proposals/fleet-multiple-abm-vpp.md

```diff
@@ -0,0 +1,131 @@
+### Multiple ABM/VPP tokens
+
+- Figma: https://www.figma.com/design/j2M1heOh8eZD6LcUJks6HE/%239956-Add-multiple-Apple-Business-Manager-and-Volume-Purchasing-Program-connections?node-id=2-130&t=3vPRYgndVgzoD8e4-0
+- Issue: https://github.com/fleetdm/fleet/issues/9956
+
+- Constraints/bussines logic:
+    - VPP tokens can be assigned to: "All teams", "No team" or a team. Default is "All teams"
+    - ABM tokens can be assigned to: "No team" or a team. Default is "No team"
+    - Multiple ABM/VPP tokens per team.
+    - Transferring hosts between teams is not allowed with > 1 token.
```

Comment:
We removed this, don't know if Noah hadn't updated the figma yet

Reply:
are you thinking about transferring hosts or multiple tokens? I left you a message about the latter in #g-mdm

Reply:
Transferring. Since we need to maintain the relationship of host -> token it is owned by transferring shouldn't affect that.

Reply:
great news, thanks! I think I was looking at "scratchpad" in Figma. Removing 🎉